GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -739,135 +743,147 @@ - (void)testNotHandleWrongPath {
 #pragma mark - Tests - disconnectWithCallback:
 
 // Verifies disconnect calls callback with no errors if access token is present.
-- (void)testDisconnect_accessToken {
+- (void)testDisconnect_accessTokenIsPresent {
   [_keychainHandler saveAuthState:_authState];
-  [[[_authState expect] andReturn:_tokenResponse] lastTokenResponse];
-  [[[_tokenResponse expect] andReturn:kAccessToken] accessToken];
-  [[[_authorization expect] andReturn:_fetcherService] fetcherService];
+  OCMStub([_authState lastTokenResponse]).andReturn(_tokenResponse);
+  OCMStub([_tokenResponse accessToken]).andReturn(kAccessToken);
+  
+  GIDHTTPFetcherTestBlock testBlock =
+      ^(NSURLRequest *request, GIDHTTPFetcherFakeResponseProviderBlock responseProvider) {
+        [self verifyRevokeRequest:request withToken:kAccessToken];
+        NSData *data = [[NSData alloc] init];
+        responseProvider(data, nil);
+      };
```

Comment:
Add an expectation to make sure the test block is actually invoked, otherwise the `verifyRevokeRequest:withToken:` call may not be made.

Same for other test cases.

Reply:
Done.